src/app/calculator/calculator.component.ts

```diff
@@ -14,65 +15,160 @@ import { MOCK } from '../shared/mockdata';
 export class CalculatorComponent {
   constructor() {
     this.motocycles = MOCK.models;
-    this.canvasLeg = new p5(this.sketch, 'canvas-human');
+    this.canvasLeg = new p5(this.sketchLeg, 'canvas-human');
     this.canvasLeg.setup = () => {
-      this.canvasLeg.createCanvas(1000, 560);
+      this.canvasLeg.createCanvas(1000, 660).position(0, 0);
       this.canvasLeg.noLoop();
     };
+    this.canvasTors = new p5(this.sketchTorsAndArm, 'canvas-human');
+    this.canvasTors.setup = () => {
+      this.canvasTors.createCanvas(1000, 660).position(0, 0);
```

Comment:
if you duplicate your code you should think to create function and move code to it. And after that call function

Reply:
ok, i think maybe use one canvas for all parts of body